docs/ruleset_functions/are_all_hvac_sys_fan_objs_autosized.md

```diff
@@ -0,0 +1,37 @@
+# are_all_hvac_sys_fan_objs_autosized  
+
+**Description:** Returns true or false. The function returns true if all fans objects associated with an hvac system are autosized. 
+
+**Inputs:**  
+- **U,P,or B-RMI**: The RMR in which the fan system object is defined. 
+- **hvac_sys_obj**: The hvac object in which the associated fan objects will be assessed.
+
+**Returns:**  
+- **are_all_hvac_sys_fan_objs_autosized**: Returns true or false. The function returns true if all fans objects associated with an hvac system are autosized.   
+ 
+**Function Call:** 
+1. get_dict_of_zones_and_terminal_units_served_by_hvac_sys()    
+
+## Logic:  
+- Create dictionary of hvac systems and associated zones and terminal units: `dict_of_zones_and_terminal_units_served_by_hvac_sys_x = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(RMI)`  
+- Set are_all_hvac_sys_fan_objs_autosized = true: `are_all_hvac_sys_fan_objs_autosized = true`  
+- Check if there is a fan system associated with the hvac system (if not then assume it is defined at the terminal unit like it is for a four-pipe fan coil unit): `if hvac.fan_system != Null:`
+    - For each supply fan associated with the fan system: `for supp_fan in hvac.fan_system.supply_fans:`   
```

Comment:
I think we should only check that supply fans are autosized.

Reply:
Updated accordingly